nn/rec.py

```diff
@@ -11,8 +11,11 @@ class LSTM(_Rec):
   """
   LSTM operating on a sequence. returns (output, final_state) tuple, where final_state is (h,c).
   """
-  def __init__(self, n_out: int, **kwargs):
-    super().__init__(n_out=n_out, unit="nativelstm2", **kwargs)
+  def __init__(self, n_out: int, zoneout: Optional[bool] = False, **kwargs):
```

Comment:
Why `Optional`? It never becomes `None`.

Reply:
had something else that was optional in mind before but dropped it, the optional stayed, will fix.